params/config.go

```diff
@@ -756,6 +757,30 @@ func (c *ChainConfig) CheckConfigForkOrder() error {
 			lastFork = cur
 		}
 	}
+
+	return nil
+}
+
+// CheckConfigBlobSchedule checks that all forks with blobs enabled explicitly
+// define the blob schedule configuration.
+func (c *ChainConfig) CheckConfigBlobSchedule() error {
```

Comment:
Why is this not in `CheckConfigForkOrder`? If we introduce another function we have to call it everywhere. This check is also kind of related to the 'fork order', so it would fit perfectly into the other function.

Reply:
Yeah I can do this - just thought as I was originally writing it there that it is a different check than checking the fork order. So either needed to change that function's name or create a new one. Could also create a new function like `CheckConfig()` that calls all methods that verify the config, but does seem a bit overkill. I don't mind adding it back to `CheckConfigForkOrder`.

Reply:
Yes please add it into `CheckConfigForkOrder`. We could change the name but it's not necessary and better if less changes.